ppr-ui/tests/unit/StaffPaymentDialog.spec.ts

```diff
@@ -76,4 +76,28 @@ describe('Payment component', () => {
     await flushPromises()
     expect(getLastEvent(wrapper, proceed)).toBe(true)
   })  
+
+  it('updates search certified when checkbox is selected', async () => {
+    expect(wrapper.vm.$store.state.stateModel.search.searchCertified).toBe(false)
+    wrapper.find('#certify-checkbox').trigger('click')
+    wrapper.findComponent(BaseDialog).vm.$emit(proceed, true)
+    await flushPromises()
+    expect(wrapper.vm.$store.state.stateModel.search.searchCertified).toBe(true)
+
+  })
+
+  it('updates store payment info', async () => {
+    wrapper.findComponent(StaffPaymentComponent).vm.$emit('update:staffPaymentData', {
+      option: 1,
+      routingSlipNumber: '999888777',
+      bcolAccountNumber: '',
+      datNumber: '',
+      folioNumber: '',
+      isPriority: false
+    })
+    await flushPromises()
+    wrapper.findComponent(BaseDialog).vm.$emit(proceed, true)
+    await flushPromises()
+    expect(wrapper.vm.$store.state.stateModel.staffPayment.routingSlipNumber).toBe('999888777')
+  })
```

Comment:
also would be good to have a test that checks the staff payment info is reset on a new one

Reply:
added a test to search bar (it's where it resets)